packages/cli/src/commands/org/resources/preview.ts

```diff
@@ -117,7 +122,7 @@ export default class Preview extends Command {
 
           Once the snapshot is created, you can preview it with the following command:
 
-            ${blueBright`coveo org:resources:preview -t ${target} -s ${snapshot.id}`}
+            ${blueBright`coveo org:resources:preview -o ${target} -s ${snapshot.id}`}
```

Comment:
Ninja fix.
I'll have to do a sanity check at some point to dynamically generate these commands instead of hard coding them

Reply:
Yeah fair enuf. Outta see tho: command are not meant to change, 'cause breaking. So prolly not really worth it